lib/LocalBinary.js

```diff
@@ -16,12 +17,29 @@ function LocalBinary(){
     this.windows = true;
     this.httpPath = 'https://s3.amazonaws.com/bstack-local-prod/BrowserStackLocal.exe';
   } else {
-    if(this.is64bits)
-      this.httpPath = 'https://s3.amazonaws.com/bstack-local-prod/BrowserStackLocal-linux-x64';
-    else
-      this.httpPath = 'https://s3.amazonaws.com/bstack-local-prod/BrowserStackLocal-linux-ia32';
+    if(this.isAlpine()) {
+      this.httpPath = 'https://s3.amazonaws.com/bstack-local-prod/BrowserStackLocal-alpine';
```

Comment:
<img width="792" alt="Screenshot 2020-08-27 at 7 18 59 PM" src="https://user-images.githubusercontent.com/59193638/91450672-2d400780-e89a-11ea-8e4e-088382e9ca36.png">
Alpine is not accessible here. I think there's a different path for that. Please check and add that.

Reply:
Its dummy for now, Bipul said he'll give link later.